src/components/forms/formUtils.js

```diff
@@ -0,0 +1,46 @@
+import {FormFieldTypes} from './forms';
```

Comment:
With the inclusion of the new `formUtils.js` file, I think we should consider moving the form behavior functions there, like form validation, and maybe `FormFieldTypes`? I know we made the forms.js file in the first place to make the options easier to identify, but it feels like it should belong in a utils XD

Reply:
Yeah! I agree. I think FormFieldTypes itself should stay in `Forms.js` because it's an easy place of reference, but moving custom validation to a `selectOptionsCustomValidation` function in utils could be a nice way to clean up the types to not be so large, and just be a central way to see how all these form components work. Will do.